File: src/server/src/main/java/com/project/infrastructure/data/OriginalWorkoutTemplateRepository.java
package com.project.infrastructure.data;

import com.project.domain.workout.OriginalWorkoutTemplate;
import io.micronaut.data.annotation.Repository;
import io.micronaut.data.repository.CrudRepository;

import java.util.UUID;

@Repository
public interface OriginalWorkoutTemplateRepository extends CrudRepository<OriginalWorkoutTemplate, UUID> {
}

File: src/server/src/main/kotlin/com/project/infrastructure/data/OriginalWorkoutTemplateRepository.kt
package com.project.infrastructure.data

import com.project.domain.workout.OriginalWorkoutTemplate
import io.micronaut.data.annotation.Repository
import io.micronaut.data.repository.CrudRepository
import java.util.UUID

@Repository
interface OriginalWorkoutTemplateRepository : CrudRepository<OriginalWorkoutTemplate, UUID>
